modules/hal_nxp/mcux/mcux-sdk-ng/CMakeLists.txt

```diff
@@ -28,10 +34,10 @@ if((${MCUX_DEVICE} MATCHES "RW61") AND (NOT DEFINED CONFIG_MINIMAL_LIBC))
   zephyr_library_sources(${ZEPHYR_CURRENT_MODULE_DIR}/mcux/mcux-sdk-ng/components/misc_utilities/fsl_memcpy.S)
 endif()
 
-add_subdirectory(middleware)
-add_subdirectory(components)
-add_subdirectory(drivers)
-add_subdirectory(device)
+include(middleware/CMakeLists.txt)
```

Comment:
The usual CMake convention is to use the .cmake extension for included files

Reply:
Hi @kesyog , yes, you are right, I updated them to .cmake. thanks.